schemas/_common-schema.json

```diff
@@ -56,6 +56,12 @@
       },
       "readOnly": true
     },
+    "source": {
+      "type": "string",
+      "description": "The source of this event",
+      "enum": ["android", "ios", "browser", "flutter", "react-native", "cordova"],
```

Comment:
Do we already know what the `cordova`/ `ionic`/`phonegap` value should be? Let's only add the known ones for now and add any new one later when it's also supported by the backend